docs/Project and Community/FAQ.rst

```diff
@@ -208,6 +210,19 @@ you.
       may apply depending on the naming method employed.
    -  Example: ``zpool create tank mirror A1 B1 mirror A2 B2``
 
+-  **/dev/disk/by-uuid/:** Not a great option
+  -   Summary: One might think from the use of "UUID" that this would
+      be an ideal option - however, in practice, this ends up listing
+      one device per **pool** ID, which is not very useful for importing
+      pools with multiple disks.
+
+-  **/dev/disk/by-partuuid/**/**by-partlabel:** Great for existing pools
+  -   Summary: Entries per partition unique identifier, great! 
```

Comment:
This wording looks a little less strict for documentation purposes, as for non-native humble me.

What do you think about:
```
-  **/dev/disk/by-partuuid/**/**by-partlabel:** Works only for existing partitions

  -   Summary: partition UUID is generated on it's creation, so usage is limited
  -   Drawbacks: you can't refer to a partition unique ID on
      an unpartitioned disk for ``zpool replace``/``add``/``attach``,
      and you can't find failed disk easily without a mapping written
      down ahead of time.
```